cherrypy/lib/static.py

```diff
@@ -37,9 +53,10 @@ def serve_file(path, content_type=None, disposition=None, name=None,
     of the 'path' argument.
 
     If disposition is not None, the Content-Disposition header will be set
-    to "<disposition>; filename=<name>". If name is None, it will be set
-    to the basename of path. If disposition is None, no Content-Disposition
-    header will be written.
+    to "<disposition>; filename=<name>; filename*=utf-8''<name>"
+    as described in RFC6266 (https://tools.ietf.org/html/rfc6266#appendix-D).
```

Comment:
We can use a sphinx role for this:
```suggestion
    as described in :rfc:`6266#appendix-D`.
```